config/locales/en.yml

```diff
@@ -1,4 +1,5 @@
 en:
+  credentials_required: Credentials required
```

Comment:
Would the translation make more sense under a new section like `webauthn`? It'll provide more context on what the translation is used for rather than putting it in the highest level of the translation file. We can probably move the session translations for the prompt into this section as well https://github.com/rubygems/rubygems.org/blob/master/app/views/multifactor_auths/mfa_prompt.html.erb

Reply:
Yeah that would also work. I figured that "Credentials required" could be used more broadly, so it made more sense to have it at a higher scope so it could be used elsewhere. That way someone who may want to use this in the future wouldn't have to move it out, or access the wrong scope.

Reply:
I think it's probably fine to leave "credentials required" at the top level, since it's so vague. Maybe we can move the MFA translations into a section, but leave this key at the top level?